Src/AutoFixtureUnitTest/RandomRangedNumberGeneratorTest.cs

```diff
@@ -312,6 +313,33 @@ public void CreationWithLimitsInBoundariesShouldReturnValueInRange(Type type, ob
             // Teardown
         }
 
+        [Theory]
+        [InlineData(typeof(byte))] 
+        [InlineData(typeof(sbyte))] 
+        [InlineData(typeof(short))]  
+        [InlineData(typeof(ushort))]
+        [InlineData(typeof(int))]  
+        [InlineData(typeof(uint))]  
+        [InlineData(typeof(long))] 
+        [InlineData(typeof(ulong))] 
+        [InlineData(typeof(decimal))]
+        [InlineData(typeof(float))]  
+        [InlineData(typeof(double))]
+        public void ShouldCorrectlyHandleRequestsWithSameMinimumAndMaximumValue(Type type)
+        {
+            // Fixture setup
+            var range = Convert.ChangeType(42, type, CultureInfo.InvariantCulture);
```

Comment:
Perhaps, we could even parameterize the `42` magic number, but not that big of a deal.

Reply:
Cannot, because no way to specify a constant for the `decimal` type.